tests/test_template_types.py

```diff
@@ -838,9 +839,9 @@ def test_WithSubjectTemplate_character_count(content, values, expected_count):
     ("((placeholder))  ", {"placeholder": "  "}, None, 0),
     ("  ", {}, None, 0),
     ("Content with ((placeholder))", {"placeholder": "something extra"}, "GDS", 33),
-    ("Just content", {}, "GDS", 12),
-    ("((placeholder))  ", {"placeholder": "  "}, "GDS", 5),
-    ("  ", {}, "GDS", 5),
+    ("Just content", {}, "GDS", 17),
+    ("((placeholder))  ", {"placeholder": "  "}, "GDS", 4),
```

Comment:
Same as test case below, I would expect us to again send `GDS: `? Note, I may be incorrect, I'm not too familiar with this stuff. If you need a second opinion then Leo is a good person to speak to as he worked on this recently.